pkg/scorecard_result.go

```diff
@@ -80,6 +81,36 @@ func (r *ScorecardResult) AsCSV(showDetails bool, logLevel zapcore.Level,
 	return nil
 }
 
+// AggregatedScore returns an aggregate score out of checker.MaxResultScore.
+func (r *ScorecardResult) AggregatedScore(checkDocs docs.Doc) (int, error) {
+	// TODO: calculate the score and make it a field
+	// of ScorecardResult
+	weights := map[string]int{"Critical": 7, "High": 5, "Medium": 3, "Low": 1}
+	// Note: aggregate score changes depending on which checks are run.
+	total := 0
```

Comment:
do these need to be float like 0.0 for total and score ?

Reply:
the current scorecard scores are integers so I made this one float for consistency. If we want more granularity, we can make the aggregate score a float.

Reply:
moved to float which gives more granularity.